README.md

```diff
@@ -144,23 +144,26 @@ Upgrading
 Significant parameter changes are listed below:
 
 * The PuppetDB module now supports PuppetDB 3.0.0 by default
-* If you want to use 5.x of the module with PuppetDB 2.x, you'll need to set `puppetdb_version => 2.y.z` or `terminus_package => 2.y.z`
-* The `puppetdb::master:puppetdb_conf` class has added a `$legacy_terminus` parameter which will be set the correct default if you set `puppetdb_version => 2.y.z` or `terminus_package => 2.y.z`.
-* The default `test_url` for the `PuppetDBConnValidator` has also been chaged to `/pdb/meta/v1/version` but will also be set the correct default if you set `puppetdb_version => 2.y.z` or `terminus_package => 2.y.z`.
+* If you want to use 5.x of the module with PuppetDB 2.x, you'll need to use the new `puppetdb::globals` class to set the version of PuppetDB you're using explicitly. The ability to configure the version has been therefore moved out of the `puppetdb` and `puppetdb::server` classes.
 For example if your config looked like this before:
 ~~~ruby
-class { 'puppetdb::master::config':
-      puppetdb_server => 'foo.example.com',
-      puppetdb_version => present,
+class {'puppetdb':
+  puppetdb_version => '2.3.5',
```

Comment:
This isn't correct for say, el7, right now you need the version string to be more like `2.3.5-1.el7`. I think debian needs the -debian1 suffix or something also.